### DISCUSSION TITLE
Fill in the grid

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1148] [Galaxy Trivia] [answered]
The number of Xs in each row and column should match the corresponding numbers for each row and column.

|       | 4 | 4 | 6 | 5 | 3 | 8 | 4 | 7 | 3 |
|-------|---|---|---|---|---|---|---|---|---|
| **6** |   |   |   |   |   |   |   |   |   |
| **5** |   |   |   |   |   |   |   |   |   |
| **5** |   |   |   |   |   |   |   |   |   |
| **5** |   |   |   |   |   |   |   |   |   |
| **8** |   |   |   |   |   |   |   |   |   |
| **1** |   |   |   |   |   |   |   |   |   |
| **4** |   |   |   |   |   |   |   |   |   |
| **5** |   |   |   |   |   |   |   |   |   |
| **5** |   |   |   |   |   |   |   |   |   |

**Comment by thuongtruong1009** [comment]
4 4 6 5 3 8 4 7 3
6 X X X X X X X X
5 X X X X X X X X
5 X X X X X X X X
5 X X X X X X X X
8 X X X X X X X X
1 X
4 X X X X
5 X X X X X
5 X X X X X

**Comment by tomerf** [comment, marked as answer]
|   | 4 | 4 | 6 | 5 | 3 | 8 | 4 | 7 | 3 |
|---|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|
| 6 | X | X | X | X |   | X |   | X |   |
| 5 |   |   | X | X |   | X |   | X | X |
| 5 |   |   | X | X | X | X |   | X |   |
| 5 |   |   | X | X |   | X |   | X | X |
| 8 | X | X | X | X |   | X | X | X | X |
| 1 |   |   |   |   | X |   |   |   |   |
| 4 | X | X |   |   |   | X | X |   |   |
| 5 |   | X | X |   |   | X | X | X |   |
| 5 | X |   |   |   | X | X | X | X |   |

**Comment by Camarota-234** [comment]
| | 4| 4| 6| 5| 3| 8| 4| 7| 3|
|-------|---|---|---|---|---|---|---|---|---|
6| X | X | X | X |  | X |  | X |  |
5|  |  | X |  | X | X | | X | X |
5|  | X | X |  |  | X |  | X | X |
5| X |  | X |  |  | X | X | X |  |
8| X | X | X | X | X | X | X | X |  |
1|  |  |  | X |  |  |  |  |  |
4| X | X |  |  | X | X |  | |  |
5|  |  | X | X |  | X | X | X |  |
5|  |  |  | X |  | X | X | X | X |

**Reply by ParanoidUser** [reply]
Well done! A little tweaking of the formatting to make it more readable.
